system/core/dynamic/HTMLHelper.cfc

```diff
@@ -1448,6 +1448,35 @@ Description :
 		</cfscript>
 	</cffunction>
 
+	<!--- elixir --->
+	<cffunction name="elixir" output="false" access="public" returntype="string" hint="Finds the versioned path for an asset">
+		<cfargument name="fileName" type="string" required="true" hint="The asset path to find relative to the includes convention directory"/>
+		<cfargument name="buildDirectory" type="string" required="false" default="build" hint="The build directory inside the includes convention directory"/>
+		<cfscript>
+			var includesConvention = controller.getSetting( "includesConvention", true );
+			var appMapping = controller.getSetting( "appMapping" );
```

Comment:
So...something like this?
```cfc
var mapping = event.getCurrentModule() != "" ?
    event.getModuleRoot() :
    controller.getSetting( "appMapping" );

var href = "#mapping#/#includesConvention#/#arguments.path#";
```
Or does the `appMapping` need to be there regardless, more like:
```cfc
var appMapping = controller.getSetting( "appMapping" );
var moduleMapping = event.getCurrentModule() != "" ?
    event.getModuleRoot() & "/" :
    "";

var href = "#appMapping#/#moduleMapping##includesConvention#/#arguments.path#";
```